README-sycl.md

```diff
@@ -1,22 +1,16 @@
 # llama.cpp for SYCL
 
-[Background](#background)
-
-[OS](#os)
-
-[Intel GPU](#intel-gpu)
-
-[Linux](#linux)
-
-[Windows](#windows)
-
-[Environment Variable](#environment-variable)
-
-[Known Issue](#known-issue)
-
-[Q&A](#q&a)
-
-[Todo](#todo)
+- [Background](#background)
+- [OS](#os)
+- [Intel GPU](#intel-gpu)
+- Build and run:
```

Comment:
remove "Build and run:"
It's not chapter here.

Move Docker, Linux, Windows in root, like [Todo].

Reply:
I thought it would be easier to follow if we divide into smaller sub-parts, but I'll keep it as before (no sub-parts). Maybe we'll decide later, when this docs has more info.

This change is reverted in https://github.com/ggerganov/llama.cpp/pull/5228/commits/708a3221f8849a35145d15daad415859bbc56130